packages/expo-modules-core/ios/Core/DynamicTypes/DynamicEncodableType.swift

```diff
@@ -0,0 +1,47 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+/**
+ Dynamic type for values conforming to `Encodable` protocol.
+ Note that currently it can only encode from native to JavaScript values, thus cannot be used for arguments.
+ */
+internal struct DynamicEncodableType: AnyDynamicType {
+  static let shared = DynamicEncodableType()
+
+  func wraps<InnerType>(_ type: InnerType.Type) -> Bool {
+    return type is Encodable
+  }
+
+  func equals(_ type: any AnyDynamicType) -> Bool {
+    // Just mocking it here as we don't really need this function and we rather want to keep it a singleton
+    return false
+  }
```

Comment:
I think we should remove these functions. As far as I know they are used only in tests 😂